### ISSUE TITLE
Installer issue

### ISSUE THREAD

**Issue by SyntaxxD** [inventree/InvenTree#3959]
### Deployment Method

- [X] Installer
- [ ] Docker Development
- [ ] Docker Production
- [ ] Bare metal Development
- [ ] Bare metal Production

### Describe the problem*

Install scirpt doesn't work on Debian 11

### Steps to Reproduce

wget -Nq https://raw.githubusercontent.com/InvenTree/InvenTree/master/contrib/install.sh && bash install.sh

### Relevant log output

```bash
### Installer for InvenTree - source: inventree/stable
### Detected distribution: Debian GNU/Linux 11
### Distribution not supported
This OS is currently not supported
please install manually using https://inventree.readthedocs.io/en/stable/start/install/
or check https://github.com/inventree/InvenTree/issues/3836 for packaging for your OS.
If you think this is a bug please file an issue at
https://github.com/inventree/InvenTree/issues/new?template=install.yaml
```

**Comment by matmair** [comment]
See #3933

**Comment by matmair** [comment]
@SyntaxxD a fix was released to `master`. Please try to run 
```
wget -Nq https://raw.githubusercontent.com/InvenTree/InvenTree/master/contrib/install.sh && bash install.sh master
```
and report back when you find time.

**Comment by matmair** [comment]
This is resolved by #3962. Closing this, for now, feel free to reopen it if the specific error was not resolved.
